VladMeleshko/nodejs-course-2021Q2
# Task7: PostgreSQL & Typeorm

1. Task: https://github.com/rolling-scopes-school/basic-nodejs-2021Q2/blob/master/descriptions/postgresql-typeorm.md
2. Done: 20.06.2021 / deadline: 20.06.2021
3. Self-test ( 180 / 180 ):
[x] В качестве источника данных для users используется PostgreSQL база данных, работа с которой происходит при помощи typeorm. ( 40 / 40 )
[x] В качестве источника данных для tasks используется PostgreSQL база данных, работа с которой происходит при помощи typeorm. ( 40 / 40 )
[x] В качестве источника данных для boards используется PostgreSQL база данных, работа с которой происходит при помощи typeorm. ( 40 / 40 )
[x] Для создания таблиц с сущностями используются миграции. ( 50 / 50 )
[x] Переменные, используемые для подключения к базе данных, хранятся в .env. ( 10 / 10 )

- Штрафов нет.
- Общая оценка - 180.

How to check project local:
- install postgress;
- verifie environment variables on .env;
- setup emty base created on .env;
- run `npm run start-migration`.

How to check project in docker:
- run `docker-compose up --build`.
